pycm/cm.py

```diff
@@ -246,62 +218,46 @@ def __str__(self):
             warn(CLASS_NUMBER_WARNING, RuntimeWarning)
         return result
 
-    def __iter__(self):
-        """
-        Iterate through confusion matrix.
-
-        :return: None
-        """
+    def __iter__(self) -> Iterator[Tuple[Any, Dict[Any, int]]]:
```

Comment:
I believe it should be `Generator` instead of `Iterator`. Am I mistaken?

Reply:
I recently found that they are the same (and `Iterator` is more general); take a look at [this link](https://stackoverflow.com/questions/2776829/difference-between-pythons-generators-and-iterators).
One of the comments says:
> Even more fun is that when type hinting a Generator that only yields values can be annotated as an [Iterator more simply](https://docs.python.org/3/library/typing.html#typing.Generator) (ie: Generator[int, None, None] === Iterator[int])